CHANGELOG.md

```diff
@@ -1,3 +1,195 @@
+# Kubermatic 2.18
+
+## [v2.18.0-rc.0](https://github.com/kubermatic/kubermatic/releases/tag/v2.18.0-rc.0)
+
+### Supported Kubernetes Versions
+
+* 1.19.0
+* 1.19.2
+* 1.19.3
+* 1.19.8
+* 1.19.9
+* 1.19.13
+* 1.20.2
+* 1.20.5
+* 1.20.9
+* 1.21.0
+* 1.21.3
+* 1.22.1
+
+### Highlights
+
+- Add Kubernetes 1.22, remove Kubernetes 1.17 and 1.18 ([#7461](https://github.com/kubermatic/kubermatic/issues/7461))
+- Add telemetry chart, use `--disable-telemetry` in installer to disable it ([#7579](https://github.com/kubermatic/kubermatic/issues/7579))
+- Metering tool integration ([#7549](https://github.com/kubermatic/kubermatic/issues/7549))
+- AWS Spot instances support ([#7073](https://github.com/kubermatic/kubermatic/issues/7073), [#3607](https://github.com/kubermatic/dashboard/issues/3607))
+
+### Breaking Changes
+
+- Kubernetes 1.19 is now the minimum supported version for master, seed and user clusters. Please upgrade all userclusters to 1.19
+  prior to upgrading to KKP 2.18.
+- Helm 2 is not supported anymore, please use Helm 3 instead. It might still be possible to install KKP using Helm 2, but it's neither
+  supported nor recommended.
+- The (in 2.14) deprecated `kubermatic` and `nodeport-proxy` Helm charts have now been removed. If you haven't done so, migrate your KKP installation
+  to use the `kubermatic-operator` Helm chart or, even better, use the KKP installer. The KKP operator will automatically manage the
+  nodeport-proxy, so the chart is not required anymore.
+- The `cert-manager` Helm chart requires admins to set the Let's Encrypt account email explicitly ([#7184](https://github.com/kubermatic/kubermatic/issues/7184))
+
```

Comment:
- The new Backup mechanism replaces the old mechanism and is not backwards compatible. Furthermore the admin of the platform needs to enable the Backup mechanism as described in the documentation. https://docs.kubermatic.com/kubermatic/master/cheat_sheets/etcd/backup-and-restore/

Reply:
Added, plus the new release notes for changes since the PR was opened. Since the etcd-launcher changes will cause all etcds to rotate, I explicitly mentioned the maxParallelReconcile option.